hack/comparealerts/main.go

```diff
@@ -0,0 +1,48 @@
+package main
+
+import (
+	"flag"
+	"os"
+
+	"comparealerts/types"
+
+	"k8s.io/klog/v2"
+)
+
+var (
+	fileUp   string
+	fileDown string
+
+	alertDiffFlagSet = flag.NewFlagSet("Alert Diff", flag.ExitOnError)
+)
+
+func init() {
+	alertDiffFlagSet.StringVar(&fileUp, "upstream-alert-file", "", "provide an upstream alert file")
+	alertDiffFlagSet.StringVar(&fileDown, "downstream-alert-file", "", "provide a downstream alert file")
+}
+
+func main() {
+	alertDiffFlagSet.Parse(os.Args[1:])
+	klog.Info("Upstream:", fileUp)
+	klog.Info("Downstream:", fileDown)
+	promRuleOne, err := types.NewPrometheusRuleFromFile(fileUp)
```

Comment:
The kind of file is not verified, that could cause some confusion, because if for example, one file is a "generic yaml" ., and the other one a right prometheus rule file , the tool will say that all the alerts in the right file are not in the first one.

I would add another additional check, something like:
```
	if promRuleOne.TypeMeta.Kind != "PrometheusRule" {
		klog.Error("Upstream file does not contain a PrometheusRule")
		return
	}
```
The same for the downstream file.